locale/en/LC_MESSAGES/django.po

```diff
@@ -3693,6 +3698,16 @@ msgstr "submitted files"
 msgid "TOTAL_POINTS"
 msgstr "Total points"
 
+#: exercise/templates/exercise/_duplicate_submission_modal.html
+msgid "DUPLICATE_SUBMISSION_TITLE"
+msgstr "Identical submission"
+
+#: exercise/templates/exercise/_duplicate_submission_modal.html
+msgid "DUPLICATE_SUBMISSION"
+msgstr ""
+"This submission is identical to the previous submission <span data-dup-"
+"submission=></span>. Are you sure you want to submit it?"
```

Comment:
This is not valid HTML. There is `=` without any `"` in the data attribute. The syntactically valid alternatives are the following:
```
<span data-dup-submission></span>
<span data-dup-submission=""></span>
```

Reply:
I have no idea where the ``"`` characters disappeared. I'll add them.

Reply:
Actually, it's probably because the translation file is already using those characters for strings. I'll remove the characters from the HTML.

Reply:
As far as I remember, django.po supports escaping too: `\"`